baseplate/lib/metrics.py

```diff
@@ -303,12 +327,26 @@ def send(self, elapsed: float, sample_rate: float = 1.0) -> None:
         :param elapsed: The elapsed time in seconds to report.
 
         """
-        serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+        # serialized = self.name + (f":{(elapsed * 1000.0):g}|ms".encode())
+        serialized = (
+            self.name + (f":{(elapsed * 1000.0):g}|ms".encode()) + _format_tags(tags).encode()
+        )
         if sample_rate < 1.0:
             sampling_info = f"@{sample_rate:g}".encode()
-            serialized = b"|".join([serialized, sampling_info])
+            # serialized = b"|".join([serialized, sampling_info])
+            serialized = b"|".join([serialized, sampling_info]) + _format_tags(tags).encode()
```

Comment:
hmm! if `format_tags` didn't add `|` itself it might be possible to have `.join()` here do that for us? worth benchmarking to see if it's worth it?

Reply:
* benchmark the ways of formatting tags for performance